### PR TITLE
Bumping up to version-3.24

### PR DESCRIPTION
**Pull request by ifranzki**

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,25 @@
++ openCryptoki 3.24
+- Add support for building Opencryptoki on the IBM AIX platform
+- Add support for the CCA token on non-IBM Z platforms (x86_64, ppc64)
+- Add support for protecting tokens with a token specific user group
+- EP11: Add support for combined CKA_EXTRACTABLE and CKA_IBM_PROTKEY_EXTRACTABLE
+- CCA: Add support for Koblitz curve secp256k1
+- CCA: Add support for IBM Dilithium (CKM_IBM_DILITHIUM)
+- CCA: Add support for RSA-OAEP with SHA224, SHA256, and SHA512 on en-/decrypt
```
Comment (jschmidb) [inline ChangeLog:8]: There is no SHA384?

File: ChangeLog [hunk 2]
```diff
@@ -1,3 +1,25 @@
...
+- ICA: Support new libica AES-GCM api using the KMA instruction on z14 and later
+- CCA/ICA/Soft/ICSF: Add support for PKCS#11 v3.0 SHA3 mechanisms
+- ICA/Soft: Add support for SHA based key derivation mechanisms
+- ICA/Soft: Add support for CKD_*_SP800 KDFs for ECDH
+- EP11/CCA/ICA/Soft: Add support for CKA_ALWAYS_AUTHENTICATE
+- EP11/CCA: Support live gest relocation for protected key (PKEY) operations
```
Comment (jschmidb) [inline ChangeLog:14]: live guest

File: ChangeLog [hunk 3]
```diff
@@ -1,3 +1,25 @@
...
+- Soft: Experimental support for IBM Dilithium via OpenSSL OQS provider
+- ICSF: Add support for SHA-2 mechanisms
+- ICSF: Performance improvements for attribute retrieval
+- p11sak: Add support for exporting a key or certificate as URI-PEM file
+- p11sak: Import/export of IBM Dilithium keys in 'oqsprovider' format PEM files
+- p11sak: Add option to show the master key verification patters of secure keys
```
Comment (jschmidb) [inline ChangeLog:20]: patterns

File: ChangeLog (hunk 1 above)
Comment (ifranzki) [reply ChangeLog:8]: typo, SHA256 was there before, new is SHA224, SHA384 and SHA512.

**Review by jschmidb — APPROVED** [review]
lgtm

**Pull request merged**